.github/workflows/lint-php.yml

```diff
@@ -13,29 +13,41 @@ on:
       - master
       - stable*
 
+permissions:
+  contents: read
+
+concurrency:
+  group: lint-php-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
 jobs:
   php-lint:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        php-versions: ["7.4", "8.0"]
+        php-versions: [ "8.0", "8.1", "8.2" ]
```

Comment:
this app still supports nextcloud 22 as per info.xml, so should also lint on 7.4 (and 7.3 🤔 )

Reply:
Or you increase the min version (including a app version bump)

Reply:
Hmm, good point. The same thing should be applied to unit tests as well (as they are starting at 8.0 too).

Reply:
Done.